src/core/ui/components/sign-in/screens/sign-in.module.css

```diff
@@ -1,10 +1,16 @@
 .loginOptions {
   margin: 8px 0;
   display: grid;
-  grid-template-columns: repeat(2, 1fr);
+  grid-template-columns: repeat(1, 1fr);
   gap: 8px;
 }
 
+@media screen and (min-width: 464px) {
```

Comment:
Why is this a different min width than the one above in sign-in layout?

Reply:
Because I thought maintaining **2 columns while it fits inside the container was our goal**. So its going to be 1 column until 464, where we change to 2 because the paddings have been reduced and then in 500 we apply the designs made by Hailey

Reply:
Please let me know if you prefer to have a less responsive approach and apply mobile design until 767 and then 2 columns

Reply:
Added a couple of comments explaining so as you suggested